aten/src/ATen/native/cpu/Vec256.h

```diff
@@ -39,10 +38,10 @@ namespace vec256 {
 template <class T> class Vec256 {
 public:
   T values[32 / sizeof(T)]; // Mimics AVX behavior
-  inline void load(const T *ptr) { 
-    std::memcpy(values, ptr, 32); 
-  for (size_t i = 0; i < size(); i++) {
-  }
+  inline void load(const T *ptr) {
+    std::memcpy(values, ptr, 32);
+    for (size_t i = 0; i < size(); i++) {
```

Comment:
naive question: why do we need this empty loop?

Reply:
Removed, thanks fmassa!